crates/bevy_solari/src/realtime/specular_gi.wgsl

```diff
@@ -31,6 +31,7 @@ fn specular_gi(@builtin(global_invocation_id) global_id: vec3<u32>) {
     }
     let surface = gpixel_resolve(textureLoad(gbuffer, global_id.xy, 0), depth, global_id.xy, view.main_pass_viewport.zw, view.world_from_clip);
 
+    let wo_unnormalized = view.world_position - surface.world_position;
```

Comment:
Should use this in the normalize too, instead of the subtract lol

Reply:
Oh whoops lol